local-macros/src/lib.rs

```diff
@@ -105,19 +105,19 @@ fn get_struct_interface_impl(
         "ReadonlyTextRange" => {
             quote! {
                 impl crate::ReadonlyTextRange for #ast_type_name {
-                    fn pos(&self) -> usize {
+                    fn pos(&self) -> isize {
```

Comment:
One of the factories was initializing nodes with "dummy" locations of `-1` so converted all of these from `usize` -> `isize`